core/src/main/protobuf/tensorflow/core/example/example.proto

```diff
@@ -0,0 +1,303 @@
+// Protocol messages for describing input data Examples for machine learning
```

Comment:
I think this proto is actually published in `org.tensorflow % proto` so it doesn't need to be copied

Reply:
I tried using the protobuf scope for those dependencies but for some reason, it was not finding them, let me try again

Reply:
yeah tried adding:
```
tensorFlowProto % "protobuf",
```

also looked at https://github.com/tensorflow/tensorflow/blob/master/tensorflow/java/maven/proto/pom.xml, and they don't seem to attach the `.proto` files to the jar